docs/development.rst

```diff
@@ -195,7 +195,9 @@ You can clone ``neurodamus-py`` and use ``spack develop`` as follows::
 
 .. note::
 
-    It is not possible to use spack develop  for neurodamus-model  or neurodamus-neocortex (and others). Instead it's important to always install them by downloading the sources from Gitlab. If one must edit those sources, the recommended workflow is to create a feature branch in the respective repository and convince Spack to use that branch.
+    It is not possible to use spack develop  for neurodamus-model  or neurodamus-neocortex (and others). 
```

Comment:
It should be `spack install neurodamus-model model=neocortex` with the current spack. 

Anyhow, there are a lot to update regarding the installation. Currently what we have in the doc is based on `bb5` practices. 
https://neurodamus.readthedocs.io/en/stable/development.html#installing-with-spack
@jamesgking , we need a big task to update the doc, if no bandwidth now, then in the next sprint.